schemas/expconf/v0/experiment.json

```diff
@@ -71,11 +71,27 @@
         "entrypoint": {
             "type": [
                 "string",
+                "array",
                 "null"
             ],
             "checks": {
-                "entrypoint must be of the form \"module.submodule:ClassName\"": {
-                    "pattern": "^[a-zA-Z0-9_.]+:[a-zA-Z0-9_]+$"
+                "entrypoint must contain a trial class of the form \"module.submodule:ClassName\"": {
+                    "contains": {
```

Comment:
I'm not actually familiar with the "contains" keyword.  What is this doing?

Reply:
it's a new-ish keyword (added in json schema draft-06 i believe) that checks if one or more of the items in an array follows specification (http://json-schema.org/draft-06/json-schema-validation.html#rfc.section.6.14)